app/controllers/trips_controller.rb

```diff
@@ -0,0 +1,61 @@
+class TripsController < ApplicationController
+  def index
+    @trips = Trip.all.order(:id)
+    if params[:search]
+      @trips = Trip.search(params[:search]).order("created_at ASC")
+    else
+      @trips = Trip.all.order('created_at ASC')
+    end
+  end
+
+  def new
+    @trip = Trip.new
+  end
+
+  def create
+    passenger = Passenger.find_by(id: params[:passenger_id])
```

Comment:
This would potentially be good logic to move into the `Trip` object so that all of the defaults would be captured there instead